internal/command/tokens/special.go

```diff
@@ -0,0 +1,138 @@
+package tokens
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/shlex"
+	"github.com/spf13/cobra"
+	"github.com/superfly/flyctl/internal/command"
+	"github.com/superfly/flyctl/internal/flag"
+	"github.com/superfly/macaroon"
+	"github.com/superfly/macaroon/flyio"
+	"github.com/superfly/macaroon/resset"
+)
+
+func newSpecial() *cobra.Command {
+	const (
+		short = "Create special attenuated Fly.io API tokens"
+		long  = `Attenuate a Fly.io API token for special purposes. The
+				token to be attenuated may either be passed in the -t argument
+				or in FLY_API_TOKEN.`
+		usage = "special"
```

Comment:
My 2¢: `fly tokens create` is already our place to shove whatever wacky token-generation stuff we want. I don't think we need a sub-folder for special tokens. Things that are really top-level concerns can be aliased (`fly tokens deploy` is an alias to `fly tokens create deploy`).

Reply:
yah, this is much better. Also it creates new tokens with arbitrary lifetimes vs. attenuating based on the login tokens flyctl holds (which have 3rd party caveats). Updated in latest commit.